README.md

```diff
@@ -21,34 +43,83 @@ npm install bold-ui --save
 yarn add bold-ui --save
 ```
 
-### Font family: IBM Plex Sans
+> IBM Plex Sans Font family
```

Comment:
since we are showing this info here, I think the website phrase about it would be nice too 

``` 
Bold was designed to use IBM Plex Sans as default font family, using the regular and bold styles from it.
```